orthanc/orthanc-anon/plugin/pixl.py

```diff
@@ -156,6 +155,29 @@ def SendViaStow(resourceId):
         orthanc.LogError("Failed to send via STOW")
 
 
+def SendViaFTPS(resourceId: str) -> None:
+    """
+    Makes a POST API call to upload the resource to a dicom-web server
+    using orthanc credentials as authorisation
+    """
+    # Query orthanc-anon for the study
+    query = f"{ORTHANC_URL}/studies/{resourceId}/archive"
+    try:
+        response_study = requests.get(query, auth=(ORTHANC_USERNAME, ORTHANC_PASSWORD), timeout=10)
+        success_code = 200
+        if response_study.status_code != success_code:
+            msg = "Could not download archive of resource '%s'"
+            raise RuntimeError(msg, resourceId)
+    except requests.exceptions.RequestException:
+        orthanc.LogError(f"Failed to query'{resourceId}'")
+
+    # get the zip content
+    zip_content = response_study.content
+    logging.info("Downloaded data: %s", zip_content)
```

Comment:
do we want to log the zip content? This'll be binary data right? Happy for you to log the resourceId